frontend/app/work_packages/controllers/dialogs/columns.js

```diff
@@ -41,7 +41,7 @@ module.exports = function($scope, $filter, columnsModal, QueryService, WorkPacka
   // Available selectable Columns
   vm.promise = QueryService.loadAvailableColumns()
     .then(function(availableColumns){
-      vm.availableColumns = availableColumns;
+      vm.availableColumns = QueryService.getAvailableUnusedColumns();
```

Comment:
It's not ignored. Read on and [you will see](https://github.com/machisuji/openproject/blob/fix/duplicate_available_columns/frontend/app/work_packages/controllers/dialogs/columns.js#L45) that it's actually still used.